### DISCUSSION TITLE
Elixir Heex and Surface UI Support

### DISCUSSION THREAD

**Discussion by clsource** [zed-industries/zed#6577] [Feature Requests]
Hello, 
As part of my daily jobs as an elixir developer, I have to deal with [Heex](https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#sigil_H/2) and [SurfaceUI](https://surface-ui.org/) files
in my Phoenix Applications. But the editor lacks proper syntax highlight.

Most of these files are HTML/JSX similar syntax.

Thanks :)

**Comment by ssaunderss** [comment]
I'll second this, I've been trying to muscle through without the syntax highlighting but have been finding myself bouncing back to VSCode for front end work

**Comment by JosephTLyons** [comment]
hey @clsource - thanks for the feedback.  We haven't been very active in Discussions, but we plan to start using Discussions more.  That said, I think we will still use issues to track interest in new features and languages.  Can you upvote these issues with a 👍 emoji?

- https://github.com/zed-industries/community/issues/725
- https://github.com/zed-industries/community/issues/726

I'm going to mark this as answered and lock it, but feel free add any thoughts on the linked issues.

**Comment by JosephTLyons** [comment]
I wanted to mention an update.  It looks like we will land some initial support for HEEX in v0.91.0.

**Comment by JosephTLyons** [comment]
Zed now has support for HEEX.
